interface/frontend/src/components/open-image/OpenDICOM.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="DICOM-container">
+    <div class="DICOM-description">{{ display }}</div>
+    <div class="DICOM" ref="DICOM"></div>
+  </div>
+</template>
+
+<script>
+  const cornerstone = require('cornerstone-core')
+
+  export default {
+    name: 'open-dicom',
+    props: {
+      view: {
+        type: Object,
+        default: {
+          type: 'DICOM',
+          prefixCS: ':/',
+          prefixUrl: '',
+          path: ''
+        }
+      }
+    },
+    data () {
+      return {
+        base64data: null
+      }
+    },
+    computed: {
+      info () {
+        return this.$axios.get(this.view.prefixUrl + this.view.path)
```

Comment:
For reference, here's async/await syntax for this:
```
async info() {
     const response = await this.$axios.get(this.view.prefixUrl + this.view.path); 
     return response.data;
}
```

Reply:
Thanks, done [here](https://github.com/concept-to-clinic/concept-to-clinic/pull/188/commits/64484a2ed46aedb1d3fdc5ff58c0cdd344289c6a#diff-70d179422138a3042c2135a282065626R30)